### PR TITLE
[postgres] application name has mirror suffix

### PR DESCRIPTION
**Pull request by heavycrystal**

### CODE REVIEW

File: flow/shared/postgres.go [hunk 1]
```diff
@@ -28,16 +28,21 @@ const (
 	POSTGRES_15 PGVersion = 150000
 )
 
-func GetPGConnectionString(pgConfig *protos.PostgresConfig) string {
+func GetPGConnectionString(pgConfig *protos.PostgresConfig, flowName string) string {
 	passwordEscaped := url.QueryEscape(pgConfig.Password)
+	applicationName := "peerdb"
+	if flowName != "" {
+		applicationName = "peerdb_" + flowName
```
Comment (iamKunalGupta) [inline flow/shared/postgres.go:35]: do we need to crop the application name to <64 chars?

Comment (iamKunalGupta) [reply flow/shared/postgres.go:35]: Also should we make this an opt-in? Like `PEERDB_APPLICATION_NAME_USES_MIRRORNAME` or smth?

Comment (heavycrystal) [reply flow/shared/postgres.go:35]: testing, looks like PG will truncate it if needed
I haven't seen many users care about `application_name` as long as it's identified as PeerDB so not in favour to add another setting

Comment (iskakaushik) [reply flow/shared/postgres.go:35]: this should definitely be opt-in

**Review by serprex — APPROVED** [review]

File: flow/shared/postgres.go (hunk 1 above)
Comment (iamKunalGupta) [reply flow/shared/postgres.go:35]: @heavycrystal can we make this opt in and finish up?

**Review by iamKunalGupta — APPROVED** [review]

**Pull request merged**